Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/PostProcessing/ViewSrg.azsli

```diff
@@ -26,6 +26,7 @@ partial ShaderResourceGroup ViewSrg
 
         // circle of confusion to screen ratio;
         float m_cocToScreenRatio;
+
```

Comment:
whitespace only change?

Reply:
Unintentional whitespace only change after resolving merge conflict, but the whitespace makes it more readable so decided to keep it instead of push another commit and kick of AR again